docs/source/overview.rst

```diff
@@ -1,10 +1,161 @@
-Getting started
-===============
+Getting started with Version 2.0
+================================
 
 Overview
 --------
 
-`DataTransferKit <https://github.com/ORNL-CEES/DataTransferKit>`_ is a
-library for multiphysics solution transfer.
+`DataTransferKit <https://github.com/ORNL-CEES/DataTransferKit>`_ is an
+open-source software library designed to provide parallel services for
+solution transfer for multiphysics simulations. DTK uses a general operator
+design to provide scalable parallel services for solution transfer between
+shared volumes and surfaces.
 
-Bla bla...
+DTK was originally developed at the University of Wisconsin - Madison as part
+of the Computational Nuclear Engineering Group (CNERG) group
+<http://cnerg.github.io> <https://github.com/CNERG> and is now actively
+developed at the Oak Ridge National Laboratory as part of the Computational
+Engineering and Energy Sciences (CEES) group <https://github.com/ORNL-CEES>.
```

Comment:
`` `Computational Engineering and Energy Sciences (CEES) group <https://github.com/ORNL-CEES>`_ ``

You may point to http://energy.ornl.gov instead of GitHub.  I'm not sure.